Makefile

```
@@ -0,0 +1,21 @@
+help:
+	@echo help sync test coverage lint fix
+
+sync:
+	uv run sync
+
+test:
+	uv run pytest -s -v
+
+coverage:
+	uv run pytest -s -v --cov=. --cov-report=html
```

our sonar-project properties expects the coverage report to be a `.xml` file.  
```suggestion
	uv run pytest -s -v --cov=. --cov-report=xml
```

---

would we want to do that outside the context of uploading to sonarcloud?

I was thinking of this as a Makefile for commands a dev would be running manually, I can see how to use a html report but what would we do with an xml one?

---

So my primary concern here is that over time we will begin to use the make commands to execute work in our CI or other automations.  If someone uses this command to run the tests in automation expecting that they will get a coverage report from sonar cloud they will be mistaken.  XML looks a lot like html, and can be viewed in a browser just like html.  It also reads in the code ver similarly to html.  

This isn't a merge blocker for me but is something we need to be cognizant of if this pr gets merged as is.

---

Yeah, agreed, let's not do that, and stop anyone from doing it doing reviews :)

The html output is very different from a xml one, I don't think you can easily confuse the two, but if anyone wants to read xml, they'll have to run it manually :)